### DISCUSSION TITLE
Teria alguma versão do jfa-go para windows?

### DISCUSSION THREAD

**Discussion by ruanbazilio12** [hrfee/jfa-go#358] [Q&A] [answered]
Teria alguma versão do jfa-go para windows?

**Comment by hrfee** [comment, marked as answer]
They exist, and have existed since the beginning in the usual places: [stable](https://github.com/hrfee/jfa-go/releases/tag/v0.5.1), [unstable](https://dl.jfa-go.com/view/hrfee/jfa-go), just choose the `TrayIcon_Windows_x86_64` versions.
